lib/ohai/plugins/linux/network.rb

```diff
@@ -41,6 +41,246 @@ def iproute2_binary_available?
     ["/sbin/ip", "/usr/bin/ip", "/bin/ip"].any? { |path| File.exist?(path) }
   end
 
+  def is_openvz?
+    ::File.directory?('/proc/vz')
+  end
+
+  def is_openvz_host?
+    is_openvz? && ::File.directory?('/proc/bc')
+  end
+
+  def extract_neighbours(family, iface, neigh_attr)
```

Comment:
Chef and chef projects are using US English. For consistency, could you use "neighbors"?